### ISSUE TITLE
Make front-end pretty

### ISSUE THREAD

**Issue by paulodiovani** [paulodiovani/hacktoberrank#8]
- [x] #3 must be done first

Add styles to make the front-end landing page pretty.

Optionally add your preferred CSS Framework (Bootstrap, Foundation, Miligram, Bulma...)

**Comment by AnnaelleS** [comment]
Hello ! Can I work on it? :)

**Comment by paulodiovani** [comment]
Of course. :)

**Comment by paulodiovani** [comment]
You may want to exchange some ideas with @arabyalhomsi, who bootstrapped the project using Vue.js.

**Comment by arabyalhomsi** [comment]
Sure! please contact me @AnnaelleS if any questions

**Comment by paulodiovani** [comment]
@AnnaelleS now we have a basic listing in the front-end. Which needs some refactor and new styles.

**Comment by AnnaelleS** [comment]
Done #33

**Comment by paulodiovani** [comment]
@AnnaelleS Pull request is merged.
Let me know if you plan to make improvements on this or if this is good enough.

IMO it would be nice to add:

- add a medal for first, second, and third places (just for fun)
  :1st_place_medal: :2nd_place_medal: :3rd_place_medal: 
- add total pull requests per user
- condense a bit more the pull requests list and add links
  the text could be just `user/repository/pull/999` format (without the whole link)
- ~~limit pull requests to three or four~~ show only the total number of pull requests per user and have a `[more]` button which expands the view
  https://getbootstrap.com/docs/4.0/components/collapse/

**Comment by paulodiovani** [comment]
New design from @kamaladenalhomsi has some good ideas and better colors, but it is incomplete.
So I'll be reverting the changes and create a new issue for a new style.
